Components/Rating/rating.css

```diff
@@ -0,0 +1,12 @@
+.rating-container input {
+  display: none;
+}
+
+.rating-container label {
+  cursor: pointer;
+  color: gold;
```

Comment:
Better to have variable names if the colors are repeated.

Reply:
Done